services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -2129,7 +2129,8 @@ public void addVersionAndStartIngestion(
         replicationMetadataVersionId,
         versionSwapDeferred,
         null,
-        repushSourceVersion);
+        repushSourceVersion,
+        getStore(clusterName, storeName).getLargestUsedRTVersionNumber());
```

Comment:
The caller of this method `CreateVersion.addVersionAndStartIngestion` is only used in the store migration. I feel that we should double check if it's correct to use the `LargestUsedRTVersionNumber` from the destination here. Can we verify its behavior with an existing or new integration test?

Reply:
I changed it use DEFAULT_RT_VERSION_NUMBER that will not change anything in the version and the migrated version just copies whatever is there in the source version.